src/Database/schema.sql

```diff
@@ -35,6 +35,18 @@ CREATE TABLE labels (
   FOREIGN KEY (book_id) REFERENCES books(id) ON DELETE CASCADE
 );
 
+CREATE TABLE authors (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  first_name VARCHAR(100),
+  last_name VARCHAR(100),
+);
+
+CREATE TABLE games (
+    id  INT,
+    multiplayer BOOLEAN,
+    last_played_at DATE,
+    FOREIGN KEY(id) REFERENCES item(id)
```

Comment:
Please add a separate foreign key to:
- reference the items table (item_id) &
- to reference the author's table (author_id)
hint: You can check out how I created the books table.